### PR TITLE
Update csp-adapter version

### PR DESCRIPTION
**Pull request by mbelur**
## Description

Update version support for csp-adapter for 2.7.10 and 2.7.11

### CODE REVIEW

**Comment by martyav** [comment]
Could we update the info just for 2.7.10 and split off the update for 2.7.11 into another PR, so that we can merge it in (and publish) the day of the release?

File: versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ In order to deploy and run the adapter successfully, you need to ensure its vers
 | v2.7.7          |     v2.0.2      |
 | v2.7.8          |     v2.0.2      |
 | v2.7.9          |     v2.0.2      |
+| v2.7.10         |     v2.0.2      |
+| v2.7.11         |     v2.0.4      |
```
Comment (martyav) [inline versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md:33]:
```suggestion
```

Rather have this go out at the same time as the other PRs related to v2.7.11

**Review by martyav — CHANGES_REQUESTED** [review]

**Comment by mbelur** [comment]
> Could we update the info just for 2.7.10 and split off the update for 2.7.11 into another PR, so that we can merge it in (and publish) the day of the release?

Just a quick clarification needed, do I submit the 2.7.10 change PR into the main branch?

**Comment by martyav** [comment]
> > Could we update the info just for 2.7.10 and split off the update for 2.7.11 into another PR, so that we can merge it in (and publish) the day of the release?
> 
> Just a quick clarification needed, do I submit the 2.7.10 change PR into the main branch?

Yes, if you do that it'll be immediately published once it's merged to main

**Comment by mbelur** [comment]
> > > Could we update the info just for 2.7.10 and split off the update for 2.7.11 into another PR, so that we can merge it in (and publish) the day of the release?
> > 
> > 
> > Just a quick clarification needed, do I submit the 2.7.10 change PR into the main branch?
> 
> Yes, if you do that it'll be immediately published once it's merged to main

Submitted a new PR 1134

**Pull request merged**